src/arviz_plots/plots/ppcplot.py

```diff
@@ -0,0 +1,529 @@
+    aggregate=False,
+    num_pp_samples=None,
+    random_seed=None,
+    plot_collection=None,
+    backend=None,
+    labeller=None,
+    aes_map=None,
+    plot_kwargs=None,
+    stats_kwargs=None,
+    pc_kwargs=None,
+):
+    """Plot prior/posterior predictive and observed values as kde, cumulative or scatter plots.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data with observed_data and prior/posterior predictive data.
+    var_names : str or list of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars : {None, “like”, “regex”}, default=None
+        If None, interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    group : str, default "posterior"
+        Group to be plotted. Note: Posterior refers to posterior-predictive, prior refers to
+        prior-predictive.
+    observed : boolean, optional
+        Whether or not to plot the observed data. Defaults to True for ``group = posterior``
+        and False for ``group = prior``.
+    observed_rug : boolean, default False
+        Whether or not to plot a rug plot of the observed data. Only valid if observed=True.
+    coords : dict, optional
+        Dictionary mapping dimensions to selected coordinates to be plotted.
+        Dimensions without a mapping specified will include all coordinates for that dimension.
+        Defaults to including all coordinates for all dimensions if None.
+    sample_dims : str or sequence of hashable, optional
+        Dimensions to loop over in plotting posterior/prior predictive.
+        Note: Dims not in sample_dims or facet_dims (below) will be reduced by default.
+        Defaults to ``rcParams["data.sample_dims"]``
+    kind : {"kde", "cumulative", "scatter"}, optional
+        How to represent the marginal density. Defaults to ``rcParams["plot.density_kind"]``.
+    facet_dims : list, optional
+        Dimensions to facet over (for which multiple plots will be generated).
+        Defaults to empty list. A warning is raised if `pc_kwargs` is also used to define
+        dims to facet over with the `cols` key and `pc_kwargs` takes precedence.
+    data_pairs : dict, optional
+        Dictionary containing relations between observed data and posterior/prior predictive data.
+        Dictionary structure:
+            * key = observed data var_name
+            * value = posterior/prior predictive var_name
+        For example, data_pairs = {'y' : 'y_hat'}.
+        If None, it will assume that the observed data and the posterior/prior predictive data
+        have the same variable name
+    aggregate: bool, optional
+        If True, predictive data will be aggregated over both sample_dims and reduce_dims.
+        Defaults to False.
```

Comment:
```suggestion
    aggregate: bool, default False
        If True, predictive data will be aggregated over both sample_dims and reduce_dims.
```

When the default is the value that is used it should be documented here directly. If it is a placeholder or if it depends on other values then use `optional` and explain the behaviour in the description